### DISCUSSION TITLE
Version 1.14.1

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#2389] [Changelog]
## Bugfix
- #2385 Some addons do not run. (prio-2)

**Comment by ItsEnderium** [comment]
In 1.14.1, could you add dark theme for the new foum icons?

**Comment by ghost** [comment]
i translated the changelog that appears in da box appears on scratch says updated scratch addons (changelog)

**Reply by ghost** [reply]
i thought i had to be in the contributers list

**Reply by WorldLanguages** [reply]
Looks like you're on it already

**Comment by williamhorning** [comment]
Next time we release, could we check the box for creating a discussion instead of manually making a discussion?
![image](https://user-images.githubusercontent.com/68466727/117451396-7b6cd680-af10-11eb-997d-7b7f6dbdf22a.png)

**Reply by apple502j** [reply]
Changelogs are written before release.

**Reply by williamhorning** [reply]
> Changelogs are written before release.

@apple502j I know that, but wouldn't make more sense to publish the discussion when it is released?

**Reply by apple502j** [reply]
This changelog is mostly for those who write the release notes (and for those who want up-to-date info), so it has to come before release.

**Reply by Hans5958** [reply]
Or alternatively, do not lock the discussions so people can talk about the update...
